book-library/script.js

```diff
@@ -31,14 +31,16 @@ function submit() {
   if (
     title.value == null ||
     title.value == "" ||
+    author.value == null ||
+    author.value == "" ||
```

Comment:
acceptance criteria dictate that in order to successfully submit, users must input the title, author, and page number. However, the code was lacking validation for the author's value